src/index.ts

```diff
@@ -3,6 +3,8 @@ import senecaSn from './seneca-sn'
 import * as Seneca from 'seneca-await'
 import {startDispatch} from './dispatch'
 import cfg from './cfg'
+import {firebase} from './process-firebase'
```

Comment:
^^ I should add, spin up new processes that want firebase but not sendgrid, for example. previously we mixed it all together